common/wuwebview/wuwebview.cpp

```diff
@@ -928,6 +928,8 @@ void WuWebView::addInputsFromXml(const char *xml)
 
 bool WuWebView::getEmbeddedArchive(StringBuffer &ret)
 {
+    if (cw->getDebugValueBool("obfuscateOutput", false))
```

Comment:
This prevents ecl being returned for existing workunits where the archive is stored.